data/json/mapgen/bank.json

```
@@ -221,6 +221,7 @@
       "computers": {
         "6": {
           "name": "High Security Consolidated Computerized Bank of the Treasury",
+          "access_denied": "ERROR!  Access denied! Unauthorized access will be met with lethal force!",
```

```suggestion
          "access_denied": "ERROR!  Access denied!  Unauthorized access will be met with lethal force!",
```
Double spaces after sentences. Same with the other messages.